server/src/main/java/com/showmetheroapes/paracord/ParacordApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.showmetheroapes.paracord;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ParacordApplication {
+
+  public static void main(String[] args) {
+    SpringApplication.run(ParacordApplication.class, args);
```

Comment:
Interesting, why the `.class`? Or is it just spring magic that we need to accept as fact?

Reply:
Because run takes a Class<?> object. So you need to reference the .class to get that value from an obejct. It holds metadata about a class (as well as some things for like classloaders and blah blah)

Reply:
Gotcha, cool beans, thanks!